### PR TITLE
Fe clean up

### PR DESCRIPTION
**Pull request by DrLazer**
a lot of reorganisation of sass for reuse

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <footer>
-    <div class="footer-strip-one">
+    <div class="footer-top">
         <a href="/">
-            <div class="logo">
+            <div class="logo__image">
                 <img src="<%= asset_pack_path 'media/images/getintoteachinglogo.svg' %>" />
-                <div class="tagline">Every Lesson shapes a life.</div>
+                <div class="logo__image__tagline">Every Lesson shapes a life.</div>
```
Comment (ethax-ross) [inline app/views/layouts/_footer.html.erb:6]: What convention are you using for class names? It looks like a bit of a mix between these:

```
logo__image__tagline
footer-bottom__logo-container
call-to-action-button
```

Comment (DrLazer) [reply app/views/layouts/_footer.html.erb:6]: BEM
hyphens are part of a block name
__ double underscores separate elements
-- double hyphens are modifiers, but there aren't that many of them

Comment (ethax-ross) [reply app/views/layouts/_footer.html.erb:6]: Ah I see, the single hyphen was throwing me but you're just using that for block/element names with multiple words in 👍

**Comment by ethax-ross** [comment]
Looks good, though CI is failing and it may be worth fixing up your commits while you're at it?

**Pull request merged**